### DISCUSSION TITLE
How can I pass a client's IP address?

### DISCUSSION THREAD

**Discussion by jhofmueller** [adnanh/webhook#669] [Q&A] [answered]
Hi,

I would loke to pass a client's IP address to the script. How can I access this information? It must be visible since I can use it in 'trigger-rule'.

Thanks!

**Comment by adnanh** [comment, marked as answer]
Hi, you can use `"pass-arguments-to-command": [{ "source": "request", "name": "remote-addr" }]`

**Reply by jhofmueller** [reply]
Thanks! Works perfectly!
